### PR TITLE
Suggested topics for Socratic 31

### PR DESCRIPTION
**Pull request by EvanWinget**
Didn't break anything into sub-topics except the split between Bitcoin and Lightning - feel free to remove any of these!

### CODE REVIEW

File: content/socratic-31.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
++++
+
+Housekeeping
+------------
+
+- Questions are encouraged, including basic ones!
+- Socratic Seminars are held under the [Chatham House Rule](https://www.chathamhouse.org/about-us/chatham-house-rule): share the information you receive, but do not reveal the identity of who said it.
+- For the privacy of other attendees, please refrain from taking photographs of other people without their permission.
+- Socratic seminars are best when the moderator can let the conversation flow, so try to keep things concrete and focused.
+- The reading list covers August 26th to September 26th.
```
Comment (casey) [inline content/socratic-31.md:13]:
```suggestion
- The reading list covers September 26th to October 30th.
```

**Comment by casey** [comment]
Dope, this is awesome. I'll do another pass tomorrow and break things up, but this is a great start. Thank you!

**Comment by casey** [comment]
By the way, if you ever feel like co-hosting a Socratic, let me know. A lot of other socratic hosts has said that it works well. We could break things up by topic or switch off.

**Comment by EvanWinget** [comment]
Thanks, Casey! You're way more knowledgeable than me and do an awesome job, so for now I'll stay in the audience and weigh in on the items I have an opinion on, but I'm happy to help share items that seemed interesting to me each month

**Pull request merged**